src/molecules/DataRow/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Tooltip from 'atoms/Tooltip';
+
+import styles from './data_row.module.scss';
+
+function DataRow(props) {
+  const {
+    className,
+    variant,
+    label,
+    value,
+    description,
+  } = props;
+
+  return (
+    <div className={classnames(styles['data-row'], styles[variant], className)}>
+      <span className={classnames(styles['label'])}>
+        { label }
+        { description && <Tooltip className={styles['tip']}>{ description }</Tooltip> }
```

Comment:
Looking at the screenshot on Clubhouse (https://app.clubhouse.io/policygenius/story/1861/datarow), it seems like a `label` can have a `tooltip` and it doesn't seem like this implementation will allow for that. Am I thinking about this wrong?